src/utils/flacdiff/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_executable(flacdiff main.cpp)
+target_link_libraries(flacdiff FLAC++-static)
```

Comment:
```suggestion
target_link_libraries(flacdiff FLAC++-static utf8)
```

Reply:
I've checked flacdiff, it depends on win_utf8_io rather than utf8